Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/source/StatisticsProvider.kt

```diff
@@ -78,6 +83,9 @@ class StatisticsProvider @Inject constructor(
         val rawData = server.getRawStatistics()
         return parser.parse(rawData).also {
             Timber.tag(TAG).d("Parsed from server: %s", it)
+            if (!it.isDataAvailable) {
+                Timber.tag(TAG).e("RawData: ${rawData.toHexString()}")
```

Comment:
and also maybe LogLevel W would be a better match.
Its not really an Error

Reply:
updated